### ISSUE TITLE
lock: tell when we start waiting for it, tell when we got it

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#136]
maybe only in verbose mode?

**Comment by ThomasWaldmann** [comment]
note for implementation: check WHERE the lock is acquired for ssh: mode backups. Is it on client or repo server?

**Comment by ThomasWaldmann** [comment]
```
$ borg init -v tw@localhost:/home/tw/w/borg/myrepo
Initializing repository at "tw@localhost:/home/tw/w/borg/myrepo"
Acquiring access to remote repository ssh://tw@localhost/home/tw/w/borg/myrepo...
Got access to remote repository ssh://tw@localhost/home/tw/w/borg/myrepo.
Encryption NOT enabled.
Use the "--encryption=repokey|keyfile|passphrase" to enable encryption.
Acquiring lock <UpgradableLock: /home/tw/.cache/borg/1754f3b532811aef49cb66b497432c5a34d2bf50d3b72bab82163c23d8a6ec78/lock tux:4047:1350170432 exclusive:True>...
Got lock <UpgradableLock: /home/tw/.cache/borg/1754f3b532811aef49cb66b497432c5a34d2bf50d3b72bab82163c23d8a6ec78/lock tux:4047:1350170432 exclusive:True>.
Synchronizing chunks cache...
Rebuilding archive collection. Known: 0 Repo: 0 Unknown: 0
Merging collection into master chunks cache...
Done.
Released lock <UpgradableLock: /home/tw/.cache/borg/1754f3b532811aef49cb66b497432c5a34d2bf50d3b72bab82163c23d8a6ec78/lock tux:4047:1350170432 exclusive:True>.
```
Almost a bit too verbose?

**Comment by anarcat** [comment]
the brackets do make it look weird, maybe a `__str__()` instead of a `__repr__()`?

**Comment by anarcat** [comment]
nevermind that, this looks fine - i think this should be closed.

**Comment by ThomasWaldmann** [comment]
Hmm, have to try again, but I think I didn't commit that change, so it still does not tell.

**Comment by anarcat** [comment]
oh right, no wonder i couldn't find the "Released lock" string...

**Comment by anarcat** [comment]
should this wait for the logging stuff first?

**Comment by ThomasWaldmann** [comment]
maybe.

**Comment by ThomasWaldmann** [comment]
I guess the proposed fix for #210 also solves this ticket.

The default lock wait time will be 1s then, so you will immediately notice if it could not get the lock (because it then terminates with a error msg that it could not get the lock).

If it could get the lock, there are other means to see it does something (e.g. the progress indication or the file listing in verbose mode).
